content/develop/get-started/vector-database.md

```diff
@@ -87,123 +94,122 @@ This quick start guide also uses the **bikes** dataset. Here is an example docum
 {{< /clients-example >}}
 
-Here is a breakdown of the `VECTOR` schema field definition:
+Here is a breakdown of the `VECTOR` field definition:
 
 * `$.description_embeddings AS vector`: The vector field's JSON path and its field alias `vector`.
-* `FLAT`: Specifies the indexing method, which is either a flat index or a hierarchical navigable small world graph (`HNSW`).
-* `TYPE FLOAT32`: Sets the type of a vector component, in this case a 32-bit floating point number.
-* `DIM 768`: The length or dimension of the embeddings, which you determined previously to be `768`.
-* `DISTANCE_METRIC COSINE`: The distance function is, in this example, [cosine similarity](https://en.wikipedia.org/wiki/Cosine_similarity).  
+* `FLAT`: Specifies the indexing method, which is either a flat index or a hierarchical navigable small world graph ([HNSW](https://arxiv.org/ftp/arxiv/papers/1603/1603.09320.pdf)).
+* `TYPE FLOAT32`: Sets the float precision of a vector component, in this case a 32-bit floating point number.
+* `DIM 768`: The length or dimension of the embeddings, determined by the chosen embedding model.
+* `DISTANCE_METRIC COSINE`: The chosen distance function: [cosine distance](https://en.wikipedia.org/wiki/Cosine_similarity).
 
 You can find further details about all these options in the [vector reference documentation]({{< relref "/develop/interact/search-and-query/advanced-concepts/vectors" >}}).
 
 ### 2. Check the state of the index
 
-As soon as you execute the [FT.CREATE]({{< baseurl >}}/commands/ft.create) command, the indexing process runs in the background. In a short time, all JSON documents should be indexed and ready to be queried. To validate that, you can use the [FT.INFO]({{< baseurl >}}/commands/ft.info) command, which provides details and statistics about the index. Of particular interest are the number of documents successfully indexed and the number of failures:  
+As soon as you execute the [FT.CREATE]({{< baseurl >}}/commands/ft.create//) command, the indexing process runs in the background. In a short time, all JSON documents should be indexed and ready to be queried. To validate that, you can use the [FT.INFO]({{< baseurl >}}/commands/ft.info//) command, which provides details and statistics about the index. Of particular interest are the number of documents successfully indexed and the number of failures:
 
 {{< clients-example search_vss validate_index >}}
 FT.INFO idx:bikes_vss
 {{< /clients-example >}}
 
-## Search and query
+## Perform vector searches
 
-This quick start guide focuses on the vector search aspect. Still, you can learn more about how to query based on vector metadata in the [document database quick start guide]({{< relref "/develop/get-started/document-database" >}}).
+This quick start guide focuses on vector search. However, you can learn more about how to query based on vector metadata in the [document database quick start guide]({{< relref "/develop/get-started/document-database" >}}).
```

Comment:
Did you mean "... you can learn more about how to query document metadata in the ..." ?

Reply:
Yep!